integration-tests/Makefile

```diff
@@ -50,5 +50,5 @@ endif
 	helm repo update
 
 .PHONY: test_smoke_product
-test_smoke_product: ## Run smoke tests for specific product ex: make test_smoke_product focus="@simulated" product="cron"
+test_smoke_product: ## Run smoke tests for specific product ex: make test_smoke_product focus="@cron" product="cron"
```

Comment:
The default behavior is to run everything with cron tag?

Reply:
Ya I originally just had it run cron by default so you have an example. I can remove it and just make it fail by default with a message if you think that makes more sense

Reply:
Why don't we keep it as it is right now a combination of cron_test and @simulated tag? I am trying to understand if focusing it on cron will run scenarios for non-simulated as well?

Reply:
`@simulated` is deprecated since the tables are all gone and we now just use the environment variable SELECTED_NETWORKS to define that which you can just pass before the make command `SELECTED_NETWORKS=SIMULATED make test_smoke_product`. The focus is still useful for this script since it can help you pinpoint a specific test instead or a whole suite like `@cron`.

Reply:
Got it. Missed that the SELECTED_NETWORKS change is already merged.